### PR TITLE
Removed dependency of nmake in windows platform and small improvements

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@conda-forge-admin , please rerender

**Comment by SylvainCorlay** [comment]
LGTM. Waiting for green checks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -61,3 +56,4 @@ extra:
     - SylvainCorlay
     - scopatz
     - wesm
+    - marcelotrevisani
```
Comment (scopatz) [inline recipe/meta.yaml:59]: Thanks for adding yourself!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 4
+  number: 5
```
Comment (SylvainCorlay) [inline recipe/meta.yaml]: I think that the build number is already 5 on the master branch.

Comment (marcelotrevisani) [reply recipe/meta.yaml]: Oh, sorry! I'm going to change that. I will just wait the build to finish.

Comment (SylvainCorlay) [reply recipe/meta.yaml]: yep, will merge right away after.

**Comment by SylvainCorlay** [comment]
It seems that you are almost there.
I am not sure that it is not required to have a `-G` section on windows?

**Comment by marcelotrevisani** [comment]
The ``-G`` is to use the ``cmake generators``, so you can generate projects to be used in visual studio, nmake and so on. In principle it's not necessary because you can just call ``cmake`` to build the project for you

*Update:*
It seems I was wrong, sorry! Actually I think cmake in conda environment tries to generate a project different than visual studio. Using the variable ``CMAKE_GENERATOR`` may fix that

**Comment by marcelotrevisani** [comment]
I'm doing some tests, because I don't have a windows machine right now

However, it seems the build is ok, the problem now is because the script do not found some compiled libraries, maybe they are in another folder or with another name

**Comment by marcelotrevisani** [comment]
@SylvainCorlay , Could you please merge that? :)

**Pull request merged**